spring-context/src/main/java/org/springframework/scheduling/support/CronSequenceGenerator.java

```diff
@@ -262,7 +274,7 @@ private void reset(Calendar calendar, List<Integer> fields) {
 	 */
 	private void parse(String expression) throws IllegalArgumentException {
 		String[] fields = StringUtils.tokenizeToStringArray(expression, " ");
```

Comment:
With the current implementation, you are tokenizing the string twice.

We would ideally prefer to avoid that.